amlb/job.py

```diff
@@ -15,6 +15,7 @@
 import threading
 import time
 
+from .resources import config as rconfig
```

Comment:
can't you find a different way to pass the param here? constructor?
I did as much as I could to keep this important low-level component fully standalone (depends only on `utils` that is itself standalone), otherwise I'm afraid we'll start to introduce various options here and make this component more fragile.

Reply:
I wasn't aware, I'll find a way around it 👍

Reply:
Thanks. Actually, that's one of those components for which I could/should write a bunch of unit tests now.

Reply:
Done.